core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/config/IndexConfiguration.java

```diff
@@ -32,6 +32,8 @@
 	@Min(1)
 	private Integer numberOfShards = 6;
 	@Min(1)
+	private Integer numberOfReplicas = 0;
```

Comment:
shouldn't it be Min(0), or numberOfReplicas = 1?

Reply:
I put Min(0) , and kept value numberOfReplicas = 0 to keep previous default value, but we could also change it too

Reply:
if we want to keep their default it should stay 0 and we will configure it to 1 through the env var, no?